experiments/tf_trainer/keras_gru_attention/run.deploy.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Edit these!
+MODEL_NAME=keras_gru_attention
+MODEL_SAVED_PATH=gs://kaggle-model-experiments/${USER}/1531159115
```

Comment:
Is the hardcoded number in the saved path intentional? Is it better to replace with a timestamp?

Reply:
Good question. This path needs to be the same as the model_dir in run.cloud.sh.
I am not sure there is an automatic way to extract it.

If we aggree that the path will always follow that pattern "gs://kaggle-model-experiments/${USER}/$TIMESTAMP" then for deployment we can take (as default) the latest timestamp.

What do you think?

Reply:
That makes sense. Though, the path we've been defaulting to has been:
"gs://kaggle-model-experiments/tf_trainer_runs/${USER}/${MODEL_NAME}/${DATETIME}/model_dir". What do you think?